### PR TITLE
Lesson1.1sql basics

### PR DESCRIPTION
**Pull request by Dan88Hus**

### CODE REVIEW

File: sql.txt [hunk 1]
```diff
@@ -1,14 +1,27 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 9 OFFSET 11;
+
```
Comment (ZipDevil) [inline sql.txt:5]: 👍

File: sql.txt [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE "A%";
+
```
Comment (ZipDevil) [inline sql.txt:9]: 👍

File: sql.txt [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity >10 ;
+
```
Comment (ZipDevil) [inline sql.txt:14]: 👍

File: sql.txt [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductName,Unit,Price,SupplierID,CategoryID) VALUES ("xyz","10 boxes",101,1,1), ("xyz","10 boxes",102,1,1), ("xyz","10 boxes",103,1,1) ;
+
```
Comment (ZipDevil) [inline sql.txt:19]: 👍 It's a good practice to have `ALL Attributes` set when you insert columns.
You can also insert just needed attributes and leave some blank (we can always come back to update them).
ex. 
`INSERT INTO Products (SupplierID, CategoryID)
VALUES
      (27, 8),
      (28, 8),
      (29, 8);`

File: sql.txt [hunk 5]
```diff
@@ -1,14 +1,27 @@
...
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products SET Price = Price + 1.50 LIMIT 2;
```
Comment (ZipDevil) [inline sql.txt:22]: `LIMIT` is not working here on the UPDATE query.
Use `WHERE` for conditions.
ex. `UPDATE Products SET Price = Price + 1.50 WHERE ProductID = 1 AND ProductID = 2;`
or  `UPDATE Products SET price = (price+1.5) WHERE ProductID <= 2;`

File: sql.txt [hunk 6]
```diff
@@ -1,14 +1,27 @@
...
+
 (6) Delete all rows of the Products Table where the price is less than 7.00. Paste your SQL statement below.
+
+DELETE FROM Products WHERE Price < 7.0
+
```
Comment (ZipDevil) [inline sql.txt:27]: 👍

**Review by ZipDevil — COMMENTED** [review]
Good job, I love one-liner SQL queries.

**Comment by Dan88Hus** [comment]
> Good job, I love one-liner SQL queries.

Thank you for checking, I love it :)

**Pull request closed without merging**